NOTICE

```diff
@@ -6,3 +6,9 @@ The Apache Software Foundation (http://www.apache.org/).
 
 The initial code for the Polaris project was donated
 to the ASF by Snowflake Inc. (https://www.snowflake.com/) copyright 2024.
+
+Apache Iceberg
+Copyright 2017-2025 The Apache Software Foundation
+
+Dremio
+Copyright 2015-2017 Dremio Corporation
```

Comment:
This addition depends on whether the code was contributed or copied as third-party content. My understanding is that it was contributed directly by Dremio and this should not be present.

Reply:
Correct, I asked the question on the other comment.

Reply:
See https://github.com/apache/polaris/pull/903#discussion_r1933538163

Reply:
@jbonofre I think we need to verify that there is no GPL code in here, given the mention of GPL code in the NOTICE updates to Iceberg.

Reply:
@rdblue yup. That's a concern to me too. Let me check.

Reply:
I checked and it's dual license: CDDL + GPL or GPL + CE.
It would be great if Nessie clarify in the NOTICE, but I don't think it's a blocker.

@rdblue thoughts ?

Reply:
I've updated Nessie NOTICE with the 0.102.5 version.